var/spack/repos/builtin/packages/sprng/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install sprng
+#
+# You can edit this file again by typing:
+#
+#     spack edit sprng
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Sprng(AutotoolsPackage):
+
+    # A list of GitHub accounts to
+    # notify when the package is updated.
+    maintainers = ['gmfricke', 'patrickb314']
+
+    homepage = "https://www.sprng.org"
+    url      = "http://www.sprng.org/Version5.0/sprng5.tar.bz2"
+
+    version('5', sha256='9172a495472cc24893e7489ce9b5654300dc60cba4430e436ce50d28eb749a66')
```

Comment:
Can you call the version `5.0` as it is on `sprng` web-site?

Reply:
If you do, you'll need a version-specific URL or `url_for_version`